### DISCUSSION TITLE
[bug]: the nav on mobile doesn't animate

### DISCUSSION THREAD

**Discussion by NiazMorshed2007** [supabase/supabase#13777] [Feature Requests]
On mobile view the nav doesn't animate at first but after clicking the cross icon it animates smoothly. It should also animate same when I click the hamburger icon to see the menu.

https://user-images.githubusercontent.com/77217706/230760174-e1200cec-2d92-4ab2-abaa-4768910b2f0b.mp4

**Comment by MildTomato** [comment]
maybe a very quick change for you @fsansalvadore

**Comment by MildTomato** [comment]
Moving to discussions as it's sort of a feature request
